packages/client/src/utils/ensureTestClientQueryEngine.ts

```diff
@@ -0,0 +1,50 @@
+import { enginesVersion, getEnginesPath } from '@prisma/engines'
+import { download } from '@prisma/fetch-engine'
+import { getNodeAPIName, Platform } from '@prisma/get-platform'
+import fs from 'fs'
+import path from 'path'
+import { ClientEngineType } from '../runtime/utils/getClientEngineType'
+/**
+ * Ensures the correct Query Engine (`node-api`/`binary`) is present. This is required as
+ * normally the downloading of the required engine is done in `getGenerators`. As the test
+ * clients bypass this we need to ensure the correct engine is present.
+ * @param clientEngineType
+ * @param platform
+ */
+export async function ensureTestClientQueryEngine(
+  clientEngineType: ClientEngineType,
+  platform: Platform,
+) {
+  const enginesPath = getEnginesPath()
+  const queryEngineLibraryPath = path.join(
+    enginesPath,
+    getNodeAPIName(platform, 'fs'),
+  )
+  const queryEngineBinaryPath = path.join(
+    enginesPath,
+    `query-engine-${platform}${platform === 'windows' ? '.exe' : ''}`,
+  )
+
+  // TMP
```

Comment:
Curious about what is this TMP comment :)

Reply:
Removed the comment